editor/components/default-block-appender/test/__snapshots__/index.js.snap

```diff
@@ -9,9 +9,9 @@ exports[`DefaultBlockAppender no block present should match snapshot 1`] = `
   <Connect(WrappedComponent) />
   <input
     className="editor-default-block-appender__content"
-    onClick={[Function]}
-    onFocus={[Function]}
-    onKeyDown={[Function]}
+    onClick={[MockFunction]}
```

Comment:
@sirreal You might like it, it is possible as of Jest 22.x to provide your own name to the mock, too. See: https://facebook.github.io/jest/blog/2017/12/18/jest-22.html#mock-function-improvements.